Shopify/hydrogen
# [DO NOT MERGE] flyio deploy example

This is an example of an Hydrogen setup for flyio

Seems there are some opinions to add this to the examples folder? vs a standalone repo. What are your thoughts?

### HOW to test your changes?
1. Follow the readme inside `examples/flyio`
2. Check that `npm run install`, `npm run dev` works
3. If possible, create a flyio account and make a deploy with `npm run deploy`
